app/controllers/concerns/goto_preview.rb

```diff
@@ -8,5 +8,4 @@ def param_form_key
   end
 
   def goto_preview? = params.dig(param_form_key, :goto_preview) == 'true'
-  def goto_provider? = params.dig(param_form_key, :goto_provider) == 'true'
```

Comment:
This is no longer needed after the refactor 91f3fa13a3c6efc513817597003e403847f3179a